packages/pluggableWidgets/rich-text-web/rollup.config.js

```diff
@@ -0,0 +1,72 @@
+import { join, extname } from "path";
+import json from "@rollup/plugin-json";
+import replace from "rollup-plugin-re";
+import command from "rollup-plugin-command";
+import copy from "recursive-copy";
+import through from "through2";
+
+export default args => {
+    const outDir = join(__dirname, "dist/tmp/widgets/ckeditor");
+    const result = args.configDefaultConfig;
+    return result.map((config, index) => {
+        // Only copying on first step is enough
+        if (index === 0) {
+            config.plugins.push(copyCKEditorDirToDist(outDir));
+        }
+        /* Need to replace icon path from bundled js files, it's step 0 and 1, 
+        tried to find much safer comparison but there are no uniqueness, so the best idea is to replace the path by index */
+        if (index === 0 || index === 1) {
+            config.plugins.push(replaceIconPath());
+        }
+        /* this step is required by sanitize-html library */
+        config.plugins.push(json());
+        return { ...config };
```

Comment:
The only reason is to keep object immutable

Reply:
I think there should be no semantic deference between `config` and `{ ...config }`, so I would stick with last one

Reply:
@raffiar can you then change to `config`?

Reply:
If it will not affect other configs then yes, because now I see `null` items in `config.plugins` array, which makes me think that object mutation is not a good idea here.